src/NHibernate/Util/ReflectHelper.cs

```diff
@@ -325,7 +325,7 @@ public static System.Type ClassForName(string name)
 		/// the method try to find the System.Type scanning all Assemblies of the <see cref="AppDomain.CurrentDomain"/>.
 		/// </remarks>
 		/// <exception cref="TypeLoadException">If no System.Type was found for <paramref name="classFullName"/>.</exception>
-				public static System.Type ClassForFullName(string classFullName)
+		public static System.Type ClassForFullName(string classFullName)
```

Comment:
Reformatting this file is quite an unrelated change but well, let it be. (Checked with ?w=1, no unexpected changes.)